--- x/qtbase/src/plugins/platforms/cocoa/qprintengine_mac_p.h
+++ y/qtbase/src/plugins/platforms/cocoa/qprintengine_mac_p.h
@@ -52,6 +52,7 @@
 //

 #include <QtCore/qglobal.h>
+#include <qpa/qplatformprintdevice.h>

 #ifndef QT_NO_PRINTER

--- x/qtbase/src/plugins/plugins.pro
+++ y/qtbase/src/plugins/plugins.pro
@@ -9,6 +9,3 @@ qtHaveModule(gui) {
     !android:qtConfig(library): SUBDIRS *= generic
 }
 qtHaveModule(widgets): SUBDIRS += styles
-
-!winrt:qtHaveModule(printsupport): \
-    SUBDIRS += printsupport

--- old/qtbase/src/tools/rcc/rcc.cpp
+++ new/qtbase/src/tools/rcc/rcc.cpp
@@ -42,6 +42,7 @@
 #include <qxmlstream.h>
 
 #include <algorithm>
+#include <iostream>
 
 #if QT_CONFIG(zstd)
 #  include <zstd.h>
@@ -233,9 +234,14 @@ void RCCFileInfo::writeDataInfo(RCCResourceLibrary &lib)
         if (sourceDate != 0)
             lastmod = sourceDate;
         static const quint64 sourceDate2 = 1000 * qgetenv("SOURCE_DATE_EPOCH").toULongLong();
-        if (sourceDate2 != 0)
+        if (sourceDate2 != 0) {
             lastmod = sourceDate2;
+            std::cerr << "----------------------------------------------------------------------------------------------------------------------" << std::endl;
+        }
         lib.writeNumber8(lastmod);
+        std::cerr << "======================================================================================================================" << std::endl;
+        std::cerr << "======================================================================================================================:" << lastmod << std::endl;
+        std::cerr << "======================================================================================================================" << std::endl;
         if (text || pass1)
             lib.writeChar('\n');
         else if (python)
